.github/workflows/test.yaml

```diff
@@ -223,6 +212,10 @@ jobs:
               export SKIP_GOFMT=1
               echo "Formatting checks will be skipped due to the use of Go version ${{ matrix.gochannel }}"
           fi
+          if [ "${{ matrix.gochannel }}" != "latest/stable" ]; then
+              export SKIP_GOLANG_CI_LINT=1
```

Comment:
Super nitpicky nitpick, feel free to ignore: I think `SKIP_GOLANGCI_LINT` would be more consistent than `SKIP_GOLANG_CI_LINT`, it's already hard enough to remember where the hyphen goes in the binary name :)